gsadjust/pyqt_models.py

```diff
@@ -1582,26 +1600,24 @@ def setData(self, index, value, role):
         """
         if role == QtCore.Qt.CheckStateRole and index.column() == 0:
             datum = self.datums[index.row()]
-            if value == QtCore.Qt.Checked:
-                datum.checked = 2
-            elif value == QtCore.Qt.Unchecked:
-                datum.checked = 0
+            datum.checked = value
```

Comment:
The values of `QtCore.Qt.Checked` and `QtCore.Qt.Unchecked` are 2 and 0 respectively anyway, so this has the same effect.